Bottom: bddcd0de3d8eb5810903d8aabe689b479df950c0
Top:    98a6dbe66728ef172a548423328109e44bb06f17
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 01:37:45 +0000

Refresh of fixup-ununitialised-memory-in

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3ab990c..5311c21 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1621,9 +1621,8 @@ GhidDrawMask (int side, BoxType * screen)
     PCB->Data->outline_valid = true;
   }
 
+  memset (&polygon, 0, sizeof (polygon));
   polygon.Clipped = PCB->Data->outline;
-  polygon.NoHoles = NULL;
-  polygon.NoHolesValid = 0;
   if (screen)
     polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
